onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -62,6 +62,44 @@ def testSetProviders(self):
             self.assertEqual(['CPUExecutionProvider'], sess.get_providers())
 
     def testSetProvidersWithOptions(self):
+        if 'TensorrtExecutionProvider' in onnxrt.get_available_providers():
+            sess = onnxrt.InferenceSession(get_name("mul_1.onnx"))
+            self.assertIn('CUDAExecutionProvider', sess.get_providers())
+
+            options = sess.get_provider_options()
+            option = options['TensorrtExecutionProvider']
+            self.assertIn('trt_device_id', option)
+            self.assertIn('trt_has_trt_options', option)
+            self.assertIn('trt_max_workspace_size', option)
+            self.assertIn('trt_fp16_enable', option)
+            self.assertIn('trt_int8_enable', option)
+            self.assertIn('trt_int8_calibration_table_name', option)
+            self.assertIn('trt_int8_use_native_calibration_table', option)
+
+            ori_max_workspace_size = option['trt_max_workspace_size']
+            new_max_workspace_size = int(ori_max_workspace_size) // 2
+            option['trt_max_workspace_size'] = new_max_workspace_size
+            calib_table_name = '/home/onnxruntime/table.flatbuffers'
```

Comment:
>/home/onnxruntime [](http://example.com/codeflow?start=32&length=17)

Hardcoded path

Reply:
For this hardcode path string, I just wanted to test whether it's been configured properly by checking the returning string.
I'll make the test code cover error situations.

Reply:
Have added more test cases. 
But we can't currently add a expecting failure test case for loading an non-existed calibration file, since we don't know whether the test machines/GPUs have int8 capability or not. If the machine doesn't support int8, TRT EP won't load the calibration file, so the expecting loading error won't occur.